docs/src/reference/grid.md

```diff
@@ -56,6 +56,21 @@ addnodeset!
 create_coloring
 ```
 
+## Mesh Reading
+`FerriteGmsh.jl` supports all defined cells with an alias in (`Ferrite.jl`)(https://github.com/Ferrite-FEM/Ferrite.jl/blob/master/src/Grid/grid.jl#L39-L54) as well as the 3D Serendipity `Cell{3,20,6}`.
```

Comment:
```suggestion
`FerriteGmsh.jl` supports all defined cells with an alias in [`Ferrite.jl`](https://github.com/Ferrite-FEM/Ferrite.jl/blob/master/src/Grid/grid.jl#L39-L54) as well as the 3D Serendipity `Cell{3,20,6}`.
```

Reply:
Would you like me to contribute with an equivalent subsection about the `FerriteMeshParser.jl`'s interface?
Currently, only the MixedDofHandler example in #448 is using this.

Reply:
I would like it. Perhaps you could do a ### heading for FerriteGmsh and FerriteMeshParser?